src/components/Chart.js

```diff
@@ -5,11 +5,12 @@ import moment from "moment";
 class Chart extends Component {
```

Comment:
as this component got rewritten, then it can be removed

Reply:
Btw. this component should work properly with MobX as it is.
Wouldn't it?

Reply:
should work with class, but for me better, and more comfortable in write is functional JS

Reply:
sure, just noted